README.md

```diff
@@ -11,9 +11,23 @@
 
 This repository contains the conceptual documentation for .NET. The [.NET documentation site](https://learn.microsoft.com/dotnet) is built from multiple repositories in addition to this one:
 
+- [ASP.NET (Core)](https://github.com/dotnet/AspNetCore.Docs)
 - [.NET Aspire](https://github.com/dotnet/docs-aspire)
 - [.NET Desktop workloads](https://github.com/dotnet/docs-desktop)
+- [.NET MAUI](https://github.com/dotnet/docs-maui)
+- [.NET Mobile](https://github.com/dotnet/docs-mobile)
```

Comment:
```suggestion
- [.NET for Android and .NET for iOS](https://github.com/dotnet/docs-mobile)
```

Reply:
Actually, this should probably be deleted or marked as internal (or something like that). The repo is private. It shouldn't be, but it is, and I doubt that's going to change anytime soon.